_studio/shared/umc/codec/vp9_dec/include/umc_vp9_frame.h

```diff
@@ -107,6 +107,8 @@ class VP9DecoderFrame
 
     uint32_t frameCountInBS;
     uint32_t currFrameInBS;
+
+    bool isDecoded;
```

Comment:
isDecoded - is state of frame, related to one frame. We can't use condition "frame in storage, so it was submitted, but not decoded", otherwise we have to make an delete from storage in async part.